FrameworkSystem/ConfigTemplate.cfg

```diff
@@ -13,11 +13,16 @@ Services
       storeHostInfo = Operator
     }
   }
+  ## ProxyManager:
```

Comment:
You can surround the doc with
```
  ##BEGIN ProxyManager
...

  ##END
```

And use it in the ProxyManagerHandler.py (see examples e.g. in TransformationSystem). What you did works the same, but it's not the "standard" way we have done it in other DIRAC systems.

Reply:
I agree that we should follow the standard, but in my opinion this view is more readable. Given that the documentation uses not only "clippings" from the configuration but also the full text of the configuration, such a record makes it easier (many words `BEGIN`, `END` complicate reading).
So what do you think this makes sense?

In the `ProxyManagerHandler.py` is already used.

Reply:
For `ConfigTemplate.cfg` files please stick to `##BEGIN xxxxxx` and `##END`

Reply:
If my arguments are not convincing, then ok.. That is, to fix it in `ConfigTemplate.cfg`, or both in `ConfigTemplate.cfg` and in `dirac.cfg`? Because in the end everything is mixed into the overall configuration https://dirac.readthedocs.io/en/latest/AdministratorGuide/Configuration/ExampleConfig.html